core/src/test/java/tachyon/worker/hierarchy/EvictStrategyTest.java

```diff
@@ -40,13 +44,21 @@
   private static final long USER_ID = 1;
   private static final long CAPACITY = 1000;
 
+  @After
+  public final void after() {
+    WorkerConf.clear();
+    System.clearProperty("tachyon.worker.evict.strategy");
+  }
+
   @Before
   public final void before() throws IOException {
     String tachyonHome =
         File.createTempFile("Tachyon", "").getAbsoluteFile() + "U" + System.currentTimeMillis();
     String workerDirFolder = tachyonHome + "/ramdisk";
     String[] dirPaths = "/dir1,/dir2,/dir3".split(",");
-    for (int i = 0; i < 3; i++) {
+    WorkerConf.clear();
+    System.setProperty("tachyon.worker.evict.strategy", "LFU");
```

Comment:
Did you mean to commit this? Is this not changing the existing lru tests to now do lfu?